backend/src/models/Article.ts

```diff
@@ -0,0 +1,30 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const articleSchema = new Schema(
+  {
+    _id: {
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
+    userId: {
+      type: String,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+  },
+  {
+    // Set timestamps on model to automatically generate
+    timestamps: true,
+  },
+);
+
+type Article = InferSchemaType<typeof articleSchema>;
+
+export default model<Article>("Article", articleSchema);
```

Comment:
Very small nitpick but its best practice to add a newline at the very end of every file. I'm not too sure the reason for this but its a git convention.